website/content/en/highlights/2023-08-15-0-32-0-upgrade-guide.md

```diff
@@ -0,0 +1,44 @@
+---
+date: "2023-08-15"
+title: "0.32 Upgrade Guide"
+description: "An upgrade guide that addresses breaking changes in 0.32.0"
+authors: ["jszwedko"]
+release: "0.32.0"
+hide_on_release_notes: false
+badges:
+  type: breaking change
+---
+
+Vector's 0.32.0 release includes **deprecations**:
+
+1. [VRL `to_timestamp` function](#deprecated-to-timestamp)
+
+and **potentially impactful changes**:
+
+1. [Upgrade of OpenSSL from 1.1.1 to 3.1.0](#openssl-upgrade)
```

Comment:
I would suggest removing the patch version from `3.1.0` because I *believe* OpenSSL is now following semantic versioning in v3. They refer to the releases as `1.1.1` and `3.1` in their [changelog](https://www.openssl.org/news/cl31.txt).

```suggestion
1. [Upgrade of OpenSSL from 1.1.1 to 3.1](#openssl-upgrade)
```